### ISSUE TITLE
Boolean properties in params are always true

### ISSUE THREAD

**Issue by spinningarrow** [teerapap/grunt-protractor-runner#130]
I have a grunt config that sets the `params` property like this:
```
options: {
  configFile: 'path/to/conf.js',
  args: {
    params: {
      someKey: 'someValue',
      otherKey: true,
      thirdKey: false
    }
  }
}
```

But when I log the values (in the `onPrepare` function in my config file, don't think that matters) like this:
```
console.log(browser.params.someKey, browser.params.otherKey, browser.params.thirdKey)
```
I get the result
```
someValue, true, true
```

Any idea what's going on?

**Comment by spinningarrow** [comment]
I added a failing test to demonstrate this behaviour here: https://github.com/spinningarrow/grunt-protractor-runner/commit/be1377e07df81c6506dacf354c5861921bd6dea3

**Comment by spinningarrow** [comment]
I did some more digging. According to [this comment on StackOverflow](http://stackoverflow.com/questions/23135649/how-can-i-use-command-line-arguments-in-angularjs-protractor#comment47453821_23137432), it seems like this is going to be difficult to support since the `params` are passed in using command line flags which only support string values.

**Comment by chrismikehogan** [comment]
+1

**Comment by teerapap** [comment]
Sorry for taking a long time. Fixed in v2.1.2
